### ISSUE TITLE
storage test run live results in different recording in different python environments

### ISSUE THREAD

**Issue by williexu** [Azure/azure-cli#5540]
---
-`test_storage_blob_incremental_copy` results in a different recording when run in Python2 vs Python3
-the python3 recording fails when played back in python2

### Environment summary

Install Method (e.g. pip, interactive script, apt-get, Docker, MSI, edge build) / CLI version (`az --version`) / OS version / Shell Type (e.g. bash, cmd.exe, Bash on Windows)  

```
```

**Comment by tjprescott** [comment]
Out of curiousity, does the Python2 recording fail when played in Python3?

**Comment by williexu** [comment]
Nope, essentially the Python2 recording has an extra request/response that is missing in Python3. During playback, python2 will expect this, whereas python3 will ignore it.
